src/chopper.cpp

```diff
@@ -65,13 +70,41 @@ int main(int argc, char const * argv[])
             chopper::input_functor{filenames, config.precomputed_files, config.k, config.window_size};
         config.hibf_config.number_of_user_bins = filenames.size();
 
-        exit_code |= chopper::layout::execute(config, filenames);
+        config.compute_sketches_timer.start();
+        seqan::hibf::sketch::compute_sketches(config.hibf_config, sketches);
+        config.compute_sketches_timer.stop();
+
+        exit_code |= chopper::layout::execute(config, filenames, sketches);
     }
     catch (std::exception const & ext)
     {
         std::cerr << "[CHOPPER ERROR] " << ext.what() << '\n';
         return -1;
     }
 
+    if (!config.disable_sketch_output)
+    {
+        if (!std::filesystem::exists(config.sketch_directory))
+            std::filesystem::create_directory(config.sketch_directory);
```

Comment:
```suggestion
            std::filesystem::create_directories(config.sketch_directory);
```

`std::filesystem::create_directory` will fail if the parent does not exist.
Also, should this be tested? Codecov complains :P